test/e2e/common/node/pod_resize.go

```diff
@@ -932,6 +932,8 @@ func doPodResizeTests(f *framework.Framework) {
 		},
 	}
 
+	timeouts := f.Timeouts
+
 	for idx := range tests {
 		tc := tests[idx]
```

Comment:
Another issue is that all subtests run in the same namespace. Running tests in separate namespaces might help too. I'm trying that in https://github.com/kubernetes/kubernetes/pull/128719/files#diff-925709f69f421bb42210b82f6e832d44fff45daedeee3accbc35ac1b5d4f49c3R1159-R1228

Reply:
I think @AnishShah your proposal will improve things and will solve kubetest2 issue. I recall i had the same failures when i tried to migrate the e2e_node CPU Manager tests to e2e/node.

```
[FAILED] A suite timeout occurred and then the following failure was recorded in the timedout node before it exited:
Context was cancelled (cause: suite timeout occurred) after 67.125s.
There is no failure as the matcher passed to Consistently has not yet failed
In [It] at: k8s.io/kubernetes/test/e2e_node/node_container_manager_test.go:128 @ 11/20/24 13:57:28.46
}
```